_source/core/packages/tesseract_collision_doc.rst

```diff
@@ -28,7 +28,7 @@ Features
 Discrete Collision Checker Example
 ==================================
 
-.. literalinclude:: /../tesseract_collision/examples/box_box_example.cpp
+.. rli:: https://raw.githubusercontent.com/ros-industrial-consortium/tesseract/master/tesseract_collision/examples/box_box_example.cpp
```

Comment:
With the volatility of `tesseract` and related packages: should these point to specific versions of these files instead of `master` versions?

Reply:
We went with adding the start and stop `// documentation:start:2:` comment to prevent frequent breaks so the code between these comments may change and the documentation would automatically be updated. Though it does not guarantee things will not break.